### ISSUE TITLE
Cant use percentage meaningfully

### ISSUE THREAD

**Issue by ahacking** [joshwlewis/unitwise#15]
Operation|to_s|Expect
---------|--------|------
Unitwise(10, '%').to_f|10.0|0.1
Unitwise(30, 'm') * Unitwise(10, '%')|300 m.%|30 m
Unitwise(30, 'm') / Unitwise(10, '%')|3 m/%|300 m
Unitwise(50,  '%') * Unitwise(50, '%')|2500 %2|0.25 (or 25%)
Unitwise(10, '%') / Unitwise(10, '%')|1 1|1.0

It seems percentages need to be converted to a dimensionless scalar before attempting any mathematical operations.

**Comment by joshwlewis** [comment]
I just released version 1.1.0 that fixes at least this case:

```ruby
(Unitwise(10, '%') / Unitwise(10, '%')).to_s # => "1"
```

**Comment by joshwlewis** [comment]
Some of your other cases here can be resolved by being a bit more explicit as to what you want:

```ruby
Unitwise(10, '%').scalar.to_f.to_s # => "0.1"
(Unitwise(30, 'm') * Unitwise(10, '%')).to_m.to_s # => "3 m"
(Unitwise(30, 'm') / Unitwise(10, '%')).to_m.to_s # => "300 m"
(Unitwise(50, '%') * Unitwise(50, '%')).scalar.to_f # => 0.25
```

Admittedly, this is a bit overkill. Unitwise and UCUM are designed to do work with units of measure, and percentages are hardly units of measure -- I'm not sure why they are in UCUM. It probably makes more sense to just do something like this:

```ruby
(Unitwise(50, 'm') * 0.25).to_s # => "12.5 m"
```

i.e. you're looking to use scalars so using a scalar (numeric) probably makes more sense.

Anyway, thanks for reporting this
